hles/transformation/src/main/scala/org/broadinstitute/monster/dap/HLESurveyTransformationLog.scala

```diff
@@ -6,7 +6,7 @@ import org.slf4j.Logger
 import ujson.Obj
 
 // a trait to capture the generic logging mechanism we'll want with case classes for the different logging levels
-abstract class HLESurveyTransformationLog {
+abstract class HLESurveyTransformationLog extends Throwable {
```

Comment:
should we add a comment here saying "don't actually throw these exceptions unless the error requires the record to be skipped outright"?

Reply:
+1 to a comment

Reply:
Also, I think this should extend Exception, not Throwable